general-concepts/slotting/text.xml

```diff
@@ -17,6 +17,13 @@ in the ebuilds. This is purely a convention; the package manager does not treat
 <c>0</c> any different from other slot values.
 </p>
 
+<p>
+"Fake slotting" where multiple SLOTs exist for the same package but they
+are mutually exclusive (i.e. block each other) are strongly discouraged, given
```

Comment:
`"Fake slotting"` is the subject of the sentence, so the verb should be singular, I think:
```suggestion
"Fake slotting" where multiple SLOTs exist for the same package but they
are mutually exclusive (i.e. block each other) is strongly discouraged, given
```